app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteListFragment.kt

```diff
@@ -136,7 +136,10 @@ open class AutocompleteListFragment : Fragment() {
         val isEnabled = !isSelectionMode()
                 || (domainList.adapter as DomainListAdapter).selection().isNotEmpty()
         removeItem?.isEnabled = isEnabled
-        removeItem?.icon?.mutate()?.alpha = if (isEnabled) SettingsFragment.ALPHA_ENABLED else SettingsFragment.ALPHA_DISABLED
+        removeItem?.icon?.mutate()?.alpha = if (isEnabled)
```

Comment:
nit: I dislike these brief, 1 line if statements becoming 4 lines. :\ Could we try:
```kotlin
removeItem?.icon?.mutate()?.alpha =
    if (isEnabled) SettingsFragment.ALPHA_ENABLED else SettingsFragment.ALPHA_DISABLED
```

or

```kotlin
removeItem?.icon?.mutate()?.alpha = if (isEnabled) SettingsFragment.ALPHA_ENABLED
    else SettingsFragment.ALPHA_DISABLED
```

or similar?

Reply:
Intellij automatically formats the second one so that `removeItem` and `else` are in the same vertical line, which is awful so I don't think it's conventional.

I went with the former.